occupi-backend/pkg/handlers/auth_handlers.go

```diff
@@ -422,9 +422,159 @@ func reverifyUsersEmail(ctx *gin.Context, appsession *models.AppSession, email s
 
 // handler for reseting a users password TODO: complete implementation
 func ResetPassword(ctx *gin.Context, appsession *models.AppSession) {
-	// this will contain reset password logic
+    var request struct {
```

Comment:
define the structure for this struct in pkg/model/request.go incase it might be needed by other functions